docs/source/operators/telemetry.rst

```diff
@@ -0,0 +1,98 @@
+Eventlogging and Telemetry
+==========================
+
+The Notebook Server can be configured to record structured events from a running server using Jupyter's `Telemetry System`_. The types of events that the Notebook Server emits are defined by `JSON schemas`_ listed below_ emitted as JSON data, defined and validated by the JSON schemas listed below.
```

Comment:
We'll only be targetting Jupyter Server with these changes so the clause '[Tt]he Notebook Server' should be replaced with 'Jupyter Server' in a few places.  (This all looks good btw - thank you!)
```suggestion
Jupyter Server can be configured to record structured events from a running server using Jupyter's `Telemetry System`_. The types of events that Jupyter Server emits are defined by `JSON schemas`_ listed below_ emitted as JSON data, defined and validated by the JSON schemas listed below.
```

Reply:
Good catch. Thanks! The PR was originally from the notebook repo I forgot to change this now that it's ported here.

Btw this PR is for the `eventlog` endpoint for client events. The one for server event is #364 which we're trying to merge first. This one depends on that one plus after we need to implement a way to identify who sends what event first, which we discussed the other day.

I'll fix this in #364.

Reply:
OK - thanks for the reminder on the client endpoint deferral.